### PR TITLE
USSD Improvements

### PR DESCRIPTION
**Pull request by enjeyw**

### CODE REVIEW

File: test/functional/app/api/test_ussd_api.py [hunk 1]
```diff
@@ -105,10 +105,10 @@ def get_session():
     assert default_transfer_account(sender).balance == (4220 - 100 - 100 - 1250)
     assert default_transfer_account(recipient).balance == (1980 + 100 + 100 + 1250)
 
-    assert len(messages) == 2
-    sent_message = messages[0]
+    assert len(messages) == 3
+    sent_message = messages[1]
```
Comment (tristanhcole) [inline test/functional/app/api/test_ussd_api.py]: what is the additional message? terms?

File: app/server/locale/general_sms.en.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+en:
+  welcome:
+    generic: Hello %{first_name}, you have been registered on Sempo! Your balance is %{balance} %{token}.
+    grassroots: Hello %{first_name} you have been registered on Sarafu Network! Your balance is %{balance} %{token}. To use dial *384*96# Safaricom or *483*46# Airtel. For help 0757628885
+  terms: By using the service, you agree to the terms and conditions at https://withsempo.com/legal.
```
[the remark this answers is not in the document]
Comment (enjeyw) [reply app/server/locale/general_sms.en.yml]: as in in the message or on our page?

File: app/server/utils/transfer_limits.py [hunk 3]
```diff
@@ -246,8 +246,8 @@ def __init__(self,
     TransferLimit('GE Liquid Token - Standard User',
                   [AGENT_OUT_PAYMENT, WITHDRAWAL], is_user_and_liquid_token, 7,
                   transfer_filter=withdrawal_or_agent_out_filter,
-                  transfer_count=1, transfer_balance_fraction=0.10),
-
+                  # transfer_count=1, transfer_balance_fraction=0.10),
```
[the remark this answers is not in the document]
Comment (enjeyw) [reply app/server/utils/transfer_limits.py]: If u remove the limit amounts, they default to None and thus prevent any exchanges at all (as desired) - nifty.

File: test/functional/app/api/test_ussd_api.py (hunk 1 above)
Comment (enjeyw) [reply test/functional/app/api/test_ussd_api.py]: yessir

File: app/server/locale/general_sms.en.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (tristanhcole) [reply app/server/locale/general_sms.en.yml]: Let's look at doing it on our page for simplicity - in a "Third Party Services" section

**Pull request merged**